ckanext/datitrentinoit/model/mapping.py

```diff
@@ -153,6 +156,11 @@ def dateformat(d):
         'Measurement unit':  metadata.get_um(),
     }
 
+    if start_date:
```

Comment:
`start_date` is always defined, since it defaults to `1970`.
The temporal coverage should not depend on it.